smarc_sensor_plugins/smarc_sensor_plugins_ros/urdf/sonar_snippets.xacro

```diff
@@ -28,7 +28,7 @@
     </collision>
   </xacro:macro>
 
-  <xacro:macro name="multibeam_intensity_sonar"
+  <xacro:macro name="side_scan_sonar"
```

Comment:
Why change this name? It means we need to change it in all AUV definitions, right?

Reply:
This we talked about some time ago. I created a new xacro macro for the "multibeam" based on the side_scan one. But the two side scans are still in the urdf